pkgs/build-support/oci-tools/default.nix

```diff
@@ -72,7 +72,7 @@
       set -o pipefail
       mkdir -p $out/rootfs/{dev,proc,sys}
       cp ${config} $out/config.json
-      xargs tar c < ${writeReferencesToFile args} | tar -xC $out/rootfs/
+      xargs tar c < ${writeReferenceClosureToFile args} | tar -xC $out/rootfs/
```

Comment:
@spacekookie How did you made it to feed `writeReferencesToFile` a list? Is `args` meant to be a list or a path?

My best guess is that the singleton were flattened into the list assigned to `exportReferencesGraph`, causing the result to be identical to that with a single path passed.